common_sleep.Rmd

```diff
@@ -200,224 +230,271 @@ lattice::xyplot(Reaction ~ Days|Sx,
+The full model describes the evolution observed in the spaghetti plot and other descriptive plots of the data:
 $$\begin{aligned}
 Y_{ij} =& \gamma_{0} + b_{0i} + \gamma_{1}* \text{Days}_{ij} &&\text{fixed effects}\\
           &+ b_{0i} + b_{1i}*\text{Days}_{ij} &&\text{random effect}\\
           &+ \epsilon_{ij} &&\text{error}
-\end{aligned}$$
+\end{aligned} \tag{4}$$
 
 $$
 \begin{cases}
 Y_{ij} &= \pi_{0i} + \pi_{1i}* \text{Days}_{ij} + \epsilon_{ij} \\
 \pi_{0i} &= \gamma_{0} + b_{0i}  \\
 \pi_{1i} &= \gamma_{1} + b_{1i}
-\end{cases}
+\end{cases} \tag{5}
 $$
 
+Underneath is the average evolution of the whole population:
+$$E(Y_{ij}) = \gamma_{0} + \gamma_{1}* \text{Days}_{ij} \tag{6}$$
 
-Following is the average evolution of the population:
-$E(Y_{ij}) = \gamma_{0} + \gamma_{1}* \text{Days}_{ij}$
+The general linear mixed model is given by:
+$$\begin{cases}
+Y_i = X_i\beta + Z_i b_i + \epsilon_i \\
+b_i \sim N(0,D)\\
+\epsilon_i \sim N(0,\Sigma_i)
+\end{cases}\tag{7}$$
 
-The general liniar mixed model is given by:
-$Y_i = X_i\beta + Z_i b_i + \epsilon_i$
-with $b_i \sim N(0,D)$, $\epsilon_i \sim N(0,\Sigma_i)$ 
-$b_1\cdots,b_N$, $\epsilon_1\cdots,\epsilon_N$ 
-
-$$Y_i \tilde{} N(X_i\beta, Z_iDZ_i'+\Sigma_i)$$
+$$Y_i \sim N(X_i\beta, Z_iDZ_i'+\Sigma_i) \tag{8}$$
 
 R uses the marginal model and our calcuation are based on that.
-Next step is to check if the values we retrieved are actually significant so does the number of days have a significant effect on the reaction time?
-We tested this with Bootstrap and Likelihood Ratio tests because th sample zise wasn't large enough(excluding Wald test). 
 
-## Testing fixed effects - with bootstrap
-```{r message=FALSE, warning=FALSE}
-pander::pander(confint(sleep.reml,
-        method    = 'boot',
-        boot.type = 'perc',
-        oldNames  = F,
-        nsim      = 500))
-```
-Confidence interval of the intercept and days does not include 0 therefore both of them have a significant effect on reaction time.
-
-## likelihood ratio test with anova
 ```{r}
-sleep.full      <- lmer(Reaction~1+Days+(1 + Days|Subject), REML = FALSE, data = sleep)
-sleep.intercept <- lmer(Reaction~1+(1|Subject), REML = FALSE, data = sleep)
-pander::pander(anova(sleep.full,sleep.intercept))
-
+summary(sleep_reml)
 ```
-We compared an intercept only model with a model that includes days as well and we concluded that adding days as covariate it improves our model significantly.
-Days have have a significant effect on the reaction time with the p-value smaller than 0.05.
-The decrease in AIC value also supports this conclusion.
+## Values of the REML model
+Based on the above model, we find the following values: 
 
-### OLS vs LMM estimates
-#plot random intercept and random slope
-
-```{r}
-D.sleep = unclass(VarCorr(sleep.reml))$Subject 
-pander::pander(D.sleep)
+$$\begin{aligned}
+\gamma_{0}  &= 251.405 \\
+\gamma_{1}  &= 10.467  \\
+\sigma_{\epsilon}^{2} &= 654.94 \\
```

Comment:
maybe mention here what the sgmas stand for. instead of the mention below gifure 3